src/test/java/com/openlattice/rehearsal/SetupTestData.kt

```diff
@@ -36,42 +36,54 @@ open class SetupTestData : MultipleAuthenticatedUsersBase() {
     companion object {
         private const val DATA_FOLDER = "data"
         private const val FLIGHT_FOLDER = "flights"
+        private const val FLIGHT_SQL = "select * from public.socrates limit 10;"
+        private const val CONFIGURATION_KEY = "example_data"
 
         init {
             MissionControl.continueAfterSuccess()
         }
 
-        @Test
-        fun doImport() {
-            importDataSet("$DATA_FOLDER/socratesA.yaml","$FLIGHT_FOLDER/testdata1.csv")
-            importDataSet("$DATA_FOLDER/socratesB.yaml","$FLIGHT_FOLDER/testdata2.csv")
-        }
         /**
-         * Import datasets via Shuttle
+         * Import datasets from csv via Shuttle
          * @param
          */
-        fun importDataSet(flightFileName: String, dataFileName: String) {
+        fun importDataSet(flightFileName: String, dataSource: String, loadFromFile: Boolean) {
             loginAs("admin")
             val tokenAdmin = AuthenticationTest.getAuthentication(authOptions).credentials
 
             val flightFile = File(Thread.currentThread().contextClassLoader.getResource(FLIGHT_FOLDER).file,
                     flightFileName).absolutePath
-            val dataFile = File(Thread.currentThread().contextClassLoader.getResource(DATA_FOLDER).file, dataFileName)
+            val dataFile = File(Thread.currentThread().contextClassLoader.getResource(DATA_FOLDER).file, dataSource)
```

Comment:
This is a bit confusing, when you call the function with loadFromFile = false, because it isn't the dataSource or file, but a configuration.

Reply:
If we will have different configurations for using shuttle with sql payload, I would suggest to have an Optional parameter for config file and for loading the dataSource/file I would do 
```
val dataFile = if(loadFromFile) {
    File(Thread.currentThread().contextClassLoader.getResource(DATA_FOLDER).file, dataSource).absolutePath
} else {
    dataSource
}
```
and doing the import like this:
```
    @Test
    fun doImport() {
        importDataSet("socratesA.yaml", "testdata1.csv", true)
        importDataSet("socratesB.yaml", "testdata2.csv", true)
        importDataSet(
                "socratesAtlas.yaml",
                "example_data", 
                false)
    }
```